support-lib/jni/Marshal.hpp

```diff
@@ -80,7 +80,11 @@ namespace djinni
 		Bool() : Primitive("java/lang/Boolean", "valueOf", "(Z)Ljava/lang/Boolean;", "booleanValue", "()Z") {}
 		friend JniClass<Bool>;
 		friend Primitive<Bool, bool, jboolean>;
-		static JniType unbox(JNIEnv* jniEnv, jmethodID method, jobject j) noexcept { return jniEnv->CallBooleanMethod(j, method); }
+		static JniType unbox(JNIEnv* jniEnv, jmethodID method, jobject j) noexcept {
+            const auto result = jniEnv->CallBooleanMethod(j, method);
```

Comment:
Yeah, not a worry for primitives.  I did an explicit move of a LocalRef in djinni_support.cpp, but in this case I just followed my default of making everything const if it can be.  I think that RVO should actually make this method neither copy nor move regardless.